server/dirstore_test.go

```diff
@@ -44,7 +44,7 @@ func require_False(t *testing.T, b bool) {
 	}
 }
 
-func require_NoError(t *testing.T, err error) {
+func require_NoError(t testing.TB, err error) {
```

Comment:
What is this? Just curious.

Reply:
Because I use createDir() in testMQTTRunServer() which I have with testing.TB because I also use it for benchmarks that have *testing.B. Using the interface testing.TB allows an helper to be used by both test and bench.

Reply:
That new?

Reply:
testing.TB interface? Has been there quite a while, but after we copied some helpers to work with both tests and bench. So moving forward, technically we should use testing.TB for helpers (at least the ones that are not embedded), but I think that is fine to still use *testing.T but then change if the needs come to use a given helper from a bench test.